rdflib/container.py

```diff
@@ -10,41 +12,39 @@
 from random import randint
 
 
-'''Default type of container is a bag'''
 class Container(object):
-    
 
-    def __init__(self, graph, uri, seq=[],typee="Bag"):
-        
+    def __init__(self, graph, uri, seq=[], typee='Bag'):
+
         self.graph = graph
         self.uri = uri or BNode()
-        self._len=0
-        self._type=typee# Bag or Seq or Alt
+        self._len = 0
+        self._type = typee  # rdf:Bag or rdf:Seq or rdf:Alt
 
         self.append_multiple(seq)
-        
-        
-        pred_uri='http://www.w3.org/1999/02/22-rdf-syntax-ns#type'
-        container_uri='http://www.w3.org/1999/02/22-rdf-syntax-ns#'+str(self._type)
+
+        pred_uri = 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type'
+        container_uri = 'http://www.w3.org/1999/02/22-rdf-syntax-ns#'
+        container_uri += str(self._type)
         container = self.uri
-        self.graph.add((container, URIRef(pred_uri), URIRef(container_uri)))#adding triple corresponding to container type
-        
+        # adding triple corresponding to container type
+        self.graph.add((container, URIRef(pred_uri), URIRef(container_uri)))

```

Comment:
You could use `from rdflib.namespace import RDF` in the import section, and then use `RDF.type` here (preferably directly in the call to `self.graph.add` to avoid declaring this once-used `pred_uri` variable), and then either use `RDF[typee]`, or (IMHO preferably) pass `RDF.Bag` in the call from `Bag`, `RDF.Seq` from `Seq` and `RDF.Alt` from `Alt`.

(Also, I *think* `type_` is preferred over `typee` (which sounds like "the thing to be typed". But that's probably nit-picking. It would be good to have a convention for the name of a type in Python though (I commonly use `rtype` in Python-code dealing with RDF... Anyone have an opinion on this?)(

Reply:
@niklasl 
I have made the required changes.